tests/test_vector2_length.py

```diff
@@ -0,0 +1,14 @@
+import ppb_vector
+import pytest
+
+@pytest.mark.parametrize("x, y, expected", [
+    (6, 8, 10),
+    (8, 6, 10),
+    (0, 0, 0),
+    (-6, -8, 10),
+    (1, 2, 2.23606797749979)
+])
+
```

Comment:
And this white space should be removed a decorator should end on the line immediately preceding the function definition.